### ISSUE TITLE
Some keys do not register on windows 10

### ISSUE THREAD

**Issue by creesch** [nut-tree/nut.js#159]
**Version**
v1.4.2

**Short overview**
Some keys that are present in the the `Key` enum do not appear to actually work on a windows 10 system. 
Some testing shows that this is at least true for  `ScrollLock` and `Pause`.

**Issue occurs on**
- [ ] Virtual machine
- [ ] Docker container
- [x] Dev/Host system

**Steps to reproduce error**

_Step 1:_
Open https://keycode.info/ 

_Step 2:_

Press scroll lock on your physical keyboard. Note that the website registers the key press.

_Step 3:_ 
Execute the following code
```javascript
setTimeout(async () => {
    await keyboard.pressKey(Key.ScrollLock);
    await keyboard.releaseKey(Key.ScrollLock);
}, 2000)
```

_Step 4:_ 
Before the timeout expires go back to the keycode website so it has focus.

_Expected result:_
Key press is registered similar as the physical key press. 

_Actual result:_
No key press is registered.

**Additional content**

I made sure to test it with other keys as well. Those register fine. 

The keyboard layout on this system was `us-international`  I also changed it to `us` itself to see if that made a difference. That was not the case.

**Comment by s1hofmann** [comment]
Hi @creesch, thanks for reaching out.

Following your steps I can confirm this behaviour on my macOS.

**Comment by s1hofmann** [comment]
Closed by #367
